### PR TITLE
chore(documentation): Add Gorgone endpoints to API Documentation

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description

This PR add missing definitions to Gorgone endpoints.

**Fixes** # MON-5789

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: doc/API/centreon-api-v2.yaml [hunk 1]
```diff
@@ -4543,3 +4607,45 @@ components:
...
+                description: "Token used to identify the command"
+              code:
+                type: number
+                format: integer
+                description: "Return code of response"
+                example: 2
+              data:
+                type: string
+                description: "Data on JSON Format"
+                example: "{\"action\":\"getthumbprint\",\"data\":{\"thumbprint\":\"xxxx_xxxxxxxxx\"},\"message\":\"ok\"}"
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:4651]: Response is a stringified json ?
If not, type is a object and define properties

Comment (jeremyjaouen) [reply doc/API/centreon-api-v2.yaml:4651]: Yes it is a stringified JSON

**Review by kduret — DISMISSED** [review]

File: doc/API/centreon-api-v2.yaml [hunk 2]
```diff
@@ -4543,3 +4607,45 @@ components:
...
+            properties:
+              creation_time:
+                type: string
+                format: date-time
+                description: "Creation of dataset on a ISO Format"
+                example: "2020-11-24T09:13:04+01:00"
+              event_time:
+                type: string
+                format: date-time
+                description: "Time when event occured on a ISO Format"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:4633]:
```suggestion
                description: "Time when event occured on an ISO Format"
```

File: doc/API/centreon-api-v2.yaml [hunk 3]
```diff
@@ -4543,3 +4607,45 @@ components:
...
+          description: "Token used to identify the command"
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              creation_time:
+                type: string
+                format: date-time
+                description: "Creation of dataset on a ISO Format"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:4628]:
```suggestion
                description: "Creation of dataset on an ISO Format"
```

**Comment by loiclau** [comment]
don t forget v2.1

**Review by kduret — DISMISSED** [review]

File: doc/API/centreon-api-v2.yaml [hunk 4]
```diff
@@ -4543,3 +4607,45 @@ components:
...
+          description: "Token used to identify the command"
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              creation_time:
+                type: string
+                format: date-time
+                description: "Creation of dataset on an ISO Format"
```
Comment (jeremyjaouen) [inline doc/API/centreon-api-v2.yaml:4628]:
```suggestion
                description: "Creation of dataset in ISO Format"
```

File: doc/API/centreon-api-v2.yaml [hunk 5]
```diff
@@ -4543,3 +4607,45 @@ components:
...
+                example: "2020-11-24T09:13:04+01:00"
+              event_time:
+                type: string
+                format: date-time
+                description: "Time when event occured on an ISO Format"
```
Comment (jeremyjaouen) [inline doc/API/centreon-api-v2.yaml:4633]:
```suggestion
                description: "Time when event occured in ISO Format"
```

**Review by sc979 — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**